src/coreclr/tools/aot/ILCompiler.Compiler/ILCompiler.Compiler.csproj

```diff
@@ -586,8 +586,16 @@
     <Compile Include="Compiler\DependencyAnalysis\Target_LoongArch64\LoongArch64ReadyToRunHelperNode.cs" />
     <Compile Include="Compiler\DependencyAnalysis\Target_LoongArch64\LoongArch64ReadyToRunGenericHelperNode.cs" />
     <Compile Include="Compiler\DependencyAnalysis\Target_LoongArch64\LoongArch64UnboxingStubNode.cs" />
+    <Compile Include="Compiler\ObjectWriter\DwarfBuilder.cs" />
+    <Compile Include="Compiler\ObjectWriter\DwarfCie.cs" />
+    <Compile Include="Compiler\ObjectWriter\DwarfEhFrame.cs" />
+    <Compile Include="Compiler\ObjectWriter\DwarfFde.cs" />
+    <Compile Include="Compiler\ObjectWriter\DwarfNative.cs" />
     <Compile Include="Compiler\ObjectWriter\LlvmObjectWriter.cs" />
```

Comment:
I know this is `[NO REVIEW]`, but this bit of feedback is perhaps more appropriate for this stage...

`LlvmObjectWriter` would soft-conflict with the downstream [`LLVMObjectWriter.cs`](https://github.com/dotnet/runtimelab/blob/feature/NativeAOT-LLVM/src/coreclr/tools/aot/ILCompiler.LLVM/CodeGen/LLVMObjectWriter.cs). It would be nicer from our downstream perspective for it to be named something more distinctive, e. g. `NativeLlvmObjectWriter`.

Reply:
Thanks for the heads-up. I will check what is done downstream and try to avoid any conflicts.

Reply:
Renamed to LegacyObjectWriter.cs for now.